app/src/main/java/io/github/jan/kex/ui/screen/exam/ExamCreateScreen.kt

```diff
@@ -108,7 +121,19 @@ fun ExamCreateScreen(
             }
         }
         Button(
-            onClick = { onCreate(subject, selectedDate!!, theme.toHtml(), type)},
+            onClick = {
+                if (selectedDate != null && subject.isNotBlank()) {
+                    onCreate(subject, selectedDate, theme.toHtml(), type)
+                }
+                else {
+                    errorScope.launch {
+                        isError = true
+                        delay(500)
```

Comment:
wo kommen eig die 500ms her?

Reply:
ich dachte diese 200ms in den funktionsparametern von dem errorfield geben die animation an? Oder ist das einfach um die variable zu resetten?

Reply:
ne das ist wielang das braucht um breit zu werden

Reply:
ok sehr danke aber jz noch die kommentare weg dann ok

Reply:
whooooooops hab ich vergessen